docs/stencil-docs/customizing-checkout/checkout-sdk-quickstart.md

```diff
@@ -15,18 +15,18 @@
 
 The Checkout JS SDK is a JavaScript library of methods for performing actions related to checkout. It includes methods for logging in a customer, adding addresses to the checkout object, and surfacing the shipping and payment methods that a merchant has configured. It’s everything you need to build your own custom checkout page on BigCommerce.
 
-The [sample checkout app](https://github.com/bigcommerce/checkout-sdk-js-example) we provide, built in React, is a great place to get started if you prefer to build within a framework. However, the SDK is framework agnostic because it's built with VanillaJS. 
+The [sample checkout app](https://github.com/bigcommerce/checkout-sdk-js-example) we provide, built in React, is a great place to get started if you prefer to develop within a framework. However, the SDK is framework agnostic because we built it with VanillaJS. 

```

Comment:
```suggestion
Our [sample checkout React app](https://github.com/bigcommerce/checkout-sdk-js-example) is a great place to get started if you prefer to develop within a framework. However, the SDK is framework agnostic because we built it with VanillaJS. 

```